### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by helloguille** [plandex-ai/plandex#95] [General]
I would like to have it removed, but not sure if removing the binary is enough.

Thanks!

**Comment by danenania** [comment]
Hi @helloguille, did you use cloud or self-hosted? To remove all local files:

In any project directory where you created a plan:

```bash
rm -rf .plandex
rm -rf .plandex-dev # only applies if you ever ran Plandex with PLANDEX_ENV=development
```

Then to remove config from `$HOME`.

```bash
rm -rf $HOME/.plandex-home
rm -rf $HOME/.plandex-home-dev # only applies if you ever ran Plandex with PLANDEX_ENV=development
```

**Comment by danenania** [comment]
@helloguille Moved this to a discussion so it's easier for others to surface. Also made a note to add something on this to usage/docs.

**Comment by ali-issa** [comment]
You should clearly state at the top of the README that creating an account is required to use this tool. Transparency matters. Personally, I uninstall any terminal app that forces sign-in or account creation before use, it’s a matter of principle, regardless of how good the tool might be. If I had been able to just install and try it without friction, and found it useful, I likely would have subscribed anyway.

For those looking to fully uninstall:

```bash
sudo rm -f /usr/local/bin/plandex \
           /usr/local/bin/pdx \
           /usr/local/bin/plandex1 \
           /usr/local/bin/pdx1 \
           $HOME/.plandex-home-v2 \
           $HOME/.plandex-home-dev-v2 \
           $HOME/.plandex-home \
           $HOME/.plandex-home-dev

rm -rf '/path/to/where/you/ran/script/plandex_install_tmp'
```

**Reply by danenania** [reply]
Hi @ali-issa, no account is required for self-hosting. Try the [local-mode quickstart](https://docs.plandex.ai/hosting/self-hosting/local-mode-quickstart).

**Comment by ali-issa** [comment]
I’ll admit I didn’t read past the install command, I just copied and pasted it. My typical flow is: install → explore → dive into the docs when I’m ready to use the tool more seriously.

To be more specific, the initial greeting was:

```
👋 Hey there!
It looks like this is your first time using Plandex on this computer.
What would you like to do?
> Start a trial on Plandex Cloud
  Sign in, accept an invite, or create an account
```

From a UX standpoint, nothing in that prompt suggested that self-hosting was possible. I assumed account creation was required, even for local use, which led me to uninstall immediately.

This is meant as constructive feedback. For users like me, the lack of clarity during setup can create friction and potentially turn away people who might otherwise find real value in the tool. A small improvement to the initial setup messaging could go a long way in retaining more users.

**Reply by danenania** [reply]
Thanks, I hear you and appreciate the feedback. I’ll try to make it clearer 👍
